cid-redirects.json

```diff
@@ -1619,6 +1619,9 @@
   "/cid/1989": "/docs/observability/aws/integrations/aws-api-gateway",
   "/cid/1990": "/docs/observability/aws/integrations/aws-application-load-balancer",
   "/cid/1991": "/docs/integrations/amazon-aws/lambda",
+  "/cid/2003": "/docs/integrations/amazon-aws/api-gateway/",
+  "/cid/2007": "/docs/integrations/amazon-aws/elastic-container-service",
+  "/cid/2034": "/docs/integrations/amazon-aws/elastic-container-service-container-insights-cloudwatch",
```

Comment:
Can you check the name again? Do we have this doc this place?

Reply:
I have checked and found cid link only for aws lambda and there were no cid number for the other paths.

Reply:
@JV0812 Thank you.